lib/rdf/datatypes/boolean.ex

```diff
@@ -5,12 +5,15 @@ defmodule RDF.Boolean do
 
   use RDF.Datatype, id: RDF.Datatype.NS.XSD.boolean
 
-  import RDF.Literal.Guards
+  import Literal.Guards
+  alias Literal
```

Comment:
This alias shouldn't be needed, since it's already defined by using `RDF.Datatype`.

Reply:
Done.